lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -146,11 +156,17 @@ export class PublicClientApplication {
      *
      * To acquire only idToken, please pass clientId as the only scope in the Authentication Parameters
      */
-    acquireTokenRedirect(request: AuthenticationParameters): void {
+    acquireTokenRedirect(request?: AuthenticationParameters): void {
```

Comment:
We very explicitly made the `request` object required in `1.x`, why is now optional?

Reply:
I removed this in a later PR, not needed.